app/services/movie_data_service.rb

```diff
@@ -14,7 +14,6 @@ module MovieDataService
   ["unwatched movies", "unwatched movies"], ["only show unwatched", "only show unwatched"],
   ["only show watched", "only show watched"], ["movies not on a list", "movies not on a list"] ]
 
-  # TODO: move these into a TMDB::MenuOptions module
```

Comment:
We talked about this at lunch and decided that we want to let this ride as we don't see a big risk in TMDB updating their genres list in a meaningful way that would break our app.